manifests/r/Resolume/Arena/7.15.0.23381/Resolume.Arena.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Resolume.Arena
+PackageVersion: 7.15.0.23381
+PackageLocale: en-US
+Publisher: Resolume B.V.
```

```suggestion
Publisher: Resolume
```
This should match Control Panel

---

Thanks for pointing this out.